app/views/insta_posts/_grid_item.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= link_to insta_user_post_path(post.insta_user, post) do %>
+  <article class='border border-slate-300 hover:border-slate-900'>
+    <% if post.image? || post.carousel_album? %>
+      <%= image_tag post.media_url %>
+    <% elsif post.video? %>
+      <%= image_tag post.thumbnail_url %>
+    <% end %>
+  </article>
+<% end %>
```

Comment:
looks like I will need to add Viewcomponent soon. the view logic is getting out of hand